ha/monitor/k8s/object_monitor.py

```diff
@@ -167,3 +172,50 @@ def run(self):
                 break
         Log.info(f"Stopping the {self.name}...")
 
+    def _is_published_alert(self, alert) -> bool:
+        """
+        Check incoming alert is already published or not.
+        If incoming alert is not found in published alerts, then
+        it is a new alert to publish. Alert will be stored like,
+        self._published_alerts = {
+            resource_name: { generation_id : alert }
+            }
+
+        Returns:
+            True if it is published already
+            False if it is a new alert
+        """
+        incoming_alert = alert.to_dict().copy()
+        incoming_resource_name = incoming_alert.get('_resource_name')
+        incoming_generation_id = incoming_alert.get('_generation_id')
```

Comment:
Also, now the alert structure is also changed. So, this get call will also change

Reply:
Can you provide the reference?

Reply:
Done.

Reply:
Thanks!